routing/missioncontrol_store.go

```diff
@@ -57,33 +84,50 @@ func newMissionControlStore(db kvdb.Backend, maxRecords int) (*missionControlSto
 		// past (see #1874).
 		c := resultsBucket.ReadCursor()
 		for k, _ := c.First(); k != nil; k, _ = c.Next() {
-			store.numRecords++
+			keys.PushBack(k)
```

Comment:
Comment above here also a little out of date? Since we're adding to our keys queue/map rather than counting.

Reply:
done